### PR TITLE
Update Runner Ami IDs for Python 3.9 Upgrade

### PR DESCRIPTION
**Pull request by peterzhuamazon**
### Description
Update Runner Ami IDs for Python 3.9 Upgrade

### Issues Resolved
https://github.com/opensearch-project/opensearch-build/issues/3351

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by peterzhuamazon** [comment]
Please DO NOT merge this until 2023/07/05 when the upgrade happens.

Thanks.

**Comment by peterzhuamazon** [comment]
Will do the ami public once we are ready.

**Review by prudhvigodithi — APPROVED** [review]

**Pull request merged**
